views/default/elgg.js.php

```diff
@@ -110,4 +110,16 @@
 
 elgg.trigger_hook('boot', 'system');
 
-require(['elgg/init', 'elgg/ready']);
\ No newline at end of file
+require(['elgg/init', 'elgg/ready']);
+
+<?php
+if (_elgg_view_may_be_altered('lightbox/settings.js', '/lightbox/settings.js.php')) {
```

Comment:
Don't include leading / in this path. We may have to use that to distinguish absolute file paths.